src/elements.h

```diff
@@ -823,6 +840,54 @@ namespace libsemigroups {
     }
   };
 
+  //! Template class for permutations.
+  //!
+  //! The value of the template parameter \p T can be used to reduce the amount
+  //! of memory required by instances of this class; see PartialTransformation
+  //! and ElementWithVectorData for more details.
+  //!
+  //! A *permutation* \f$f\f$ is an injective transformation defined on the
+  //! whole of \f$\{0, 1, \ldots, n - 1\}\f$ for some integer \f$n\f$ called the
+  //! *degree* of \f$f\f$.
+  //! A permutation is stored as a vector of the images of
+  //! \f$\{0, 1, \ldots, n - 1\}\f$,
+  //! i.e. \f$\{(0)f, (1)f, \ldots, (n - 1)f\}\f$.
```

Comment:
I'd use round brackets here not curly ones, since the order of these images matters.